packages/server/.npmignore

```diff
@@ -1,6 +1,7 @@
 *
 # Include the package.json files that give us deep imports.
 !**/package.json
+# TODO(AS4): glasser isn't this covered by the * above?
```

Comment:
hmm, the previous line will pull `node_modules/foo/package.json` back in, right?